apps/shipments/models.py

```diff
@@ -341,3 +343,98 @@ def update_vault_hash(self, vault_hash, rate_limit=False):
     VALID_UNTIL = 24
     FUNDING_TYPE = FundingType.SHIP.value
     SHIPMENT_AMOUNT = 1
+
+
+class TrackingData(models.Model):
+    id = models.CharField(primary_key=True, default=random_id, max_length=36)
+    created_at = models.DateTimeField(auto_now_add=True)
+    device_id = models.CharField(blank=True, null=True, max_length=36)
```

Comment:
device_id should not be nullable or blank. In fact, you could probably make this a ForeignKey to `Device`. If we are receiving tracking data, then it needs to be mapped to a shipment that is already in our DB (which means the device is also already in our DB).